redfish-core/lib/pcie.hpp

```diff
@@ -572,6 +573,15 @@ inline void requestRoutesSystemPCIeDevice(App& app)
                             }
                         }
 
+                        if (size_t* property = std::get_if<size_t>(
+                                &pcieDevProperties["LanesInUse"]);
+                            property)
+                        {
+                            asyncResp->res
+                                .jsonValue["PCIeInterface"]["LanesInUse"] =
+                                *property;
+                        }
```

Comment:
you could throw an error here if you needed.

Reply:
Well you could do 

                                        const size_t* value =
                                            std::get_if<size_t>(
                                                &property.second);
                                        if (value == nullptr)
                                        {
                                            messages::internalError(
                                                asyncResp->res);
                                            return;
                                        }
                                        
   Like you did below

Reply:
Done.